internal/cli/main.go

```diff
@@ -167,12 +216,20 @@ Database drivers: `+strings.Join(database.List(), ", ")+"\n")
 		}
 
 	case "up":
+		upSet, helpPtr := newFlagSetWithHelp("up")
+
+		if err := upSet.Parse(args); err != nil {
+			log.fatalErr(err)
+		}
+
+		handleSubCmdHelp(*helpPtr, upUsage, upSet)
+
 		if migraterErr != nil {
 			log.fatalErr(migraterErr)
 		}
 
 		limit := -1
-		if flag.Arg(1) != "" {
+		if flag.NArg() > 0 {
```

Comment:
This breaks the CLI using `migrate up` (sans limit argument) because `flag.NArg()` is always > 0 because `flag.Arg(0)` is always `up`

Reply:
See: https://github.com/golang-migrate/migrate/issues/467